.github/ISSUE_TEMPLATE/--feature_request.md

```diff
@@ -0,0 +1,33 @@
+---
+name: "Feature request"
+about: Suggest an idea for this project
+title: ''
+labels: new feature
+assignees: ''
+
+---
+
+<!-- Please fill out as much of the template as you can. Delete sections if unneccesary -->
+<!-- 
+Make Sure you've checked out Existing Issues
+-->
+
+## Short Description
+<!-- Describe Your Idea -->
+
+
+## **How This Feature will help you/others**
+
+
+## **Possible Solution/Implementation Details**
+<!-- Describe the solution you'd like. -->
+
+
+## **Example/Links if Any**
+<!-- Link to already similar features if possible -->
+
+## **Can you help with this Feature**
```

Comment:
This can be changed to `Would you like to implement this?`

Reply:
I think it is fine as in commented section I have added `Pull Requests are much Appreciated`.